ci/tests/e2e-ansible.sh

```diff
@@ -58,7 +58,23 @@ test_operator() {
         kubectl logs deployment/memcached-operator -c operator
         kubectl logs deployment/memcached-operator -c ansible
         exit 1
+
+    # verify that metrics service was created
+    if ! timeout 20s bash -c -- "until kubectl get service/memcached-operator-metrics > /dev/null 2>&1; do sleep 1; done";
+    then
+        echo "Failed to get metrics service"
+        kubectl logs deployment/memcached-operator
+        exit 1
+    fi
+
+    # verify that the metrics endpoint exists
+    if ! timeout 1m bash -c -- "until kubectl run -it --rm --restart=Never test-metrics --image=registry.access.redhat.com/ubi7/ubi-minimal:latest -- curl -sfo /dev/null http://memcached-operator-metrics:8383/metrics; do sleep 1; done";
```

Comment:
I believe you asked on slack how to test custom resource metrics. 

So this is for the controller-runtime metrics, you could add the same test for the custom resource metrics, so seeing if the metrics exists on 8686 port.

Another thing you could do is, we know exactly what the metrics data will be so you could curl the endpoint and compare the result to a predefined string. If you look at the go e2e test you can see how the metrics data would look like here https://github.com/operator-framework/operator-sdk/blob/d6e1aba9fa512ff9d441b648f548399c86929643/test/e2e/_incluster-test-code/memcached_test.go#L392-L395

Hope that makes sense, otherwise feel free to ask any questions!

Reply:
Note that the verification needs to happen after line 79, so after creating CR instance. As that is when the metric should be 1.

Reply:
I think we wanted to add the same check for the other metrics port so port 8686.

Reply:
I believe I was able to manage it the check for port 8686 and for the CR info. Not having much experience with these metrics I just used grep. Let me know what you think. Thank you for the help.